doc/lsp.txt

```diff
@@ -202,6 +202,8 @@ following optional items:
     autoHighlightDiags	Automatically place signs on the lines with a
 			diagnostic message from the LSP server. By default
 			this is set to true.
+    keepFocusInReferences	Keep focus on quickfix window after ShowReferences
```

Comment:
The location list window is used to display the references. Can you change this to "Keep the focus in the location list window after |LspShowReferences| instead of moving the cursor back to the previous window."?